yeonjae/baekjoon/수학/4153-직각삼각형.js

```diff
@@ -0,0 +1,35 @@
+const { defaultMaxListeners } = require("events");
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  let result = "";
+
+  for (let i = 0; i < input.length - 1; i++) {
+    const [a, b, c] = input[i]
+      .split(" ")
+      .map(Number)
+      .sort((a, b) => a - b);
+
+    //피타고라스 정리
+    if (a ** 2 + b ** 2 === c ** 2) {
+      result += "right\n";
+    } else {
+      result += "wrong\n";
+    }
+  }
+
+  console.log(result);
```

Comment:
```suggestion
  let result = "";

  for (let i = 0; i < input.length - 1; i++) {
    const [a, b, c] = input[i]
      .split(" ")
      .map(Number)
      .sort((a, b) => a - b);

    //피타고라스 정리
    const isTriangle = c ** 2 === a ** 2 + b** 2;
    
    result += isTriangle ? `right\n` : `wrong\n`;
  }

  console.log(result);
```
- 풀이 잘봤습니다 🙇‍♂️
- if문의 조건식은 뭔가 복잡해보이면 변수로 뺴주세요 ~
- 또한 변수로 빼면 리팩토링 코드처럼 삼항 연산자로 깔끔하게 처리가 가능합니다 🙆‍♂️